sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/nullFunctions.scala

```diff
@@ -25,13 +25,15 @@ import org.apache.spark.sql.types._
 
 
 /**
- * An expression that is evaluated to the first non-null input.
+ * An expression that is evaluated to the first non-null and non-NaN input.
```

Comment:
I change this here because fillna use coalesce to replace null into default value.